arrow-libs/optics/arrow-optics-ksp-plugin/src/test/kotlin/arrow/optics/plugin/LensTests.kt

```diff
@@ -130,4 +130,23 @@ class LensTests {
       |}
       """.compilationSucceeds()
   }
+
+  @Test
+  fun `Lens for sealed class property`() {
```

Comment:
I am missing three other cases for test:
1. More than one class extending the sealed one,
2. Data classes defined outside of the sealed class,
3. Sealed class with zero inheritors.